### PR TITLE
SEARCH 2995 editting the docs page for backup as location was removed

### PR DESCRIPTION
**Pull request by klalia**

### CODE REVIEW

File: search-services/latest/admin/index.md [hunk 1]
```diff
@@ -6,11 +6,27 @@ title: Overview
 
 There are a number of ways to back up Search Services. You can set the Solr indexes backup properties either by:
 
-* Using the Admin Console **Note:** This feature is only available when you are using Alfresco Content Services Enterprise.
+* Editting the 'solrcore.properties' file
```
Comment (eliaporciani) [inline search-services/latest/admin/index.md:9]: There is a typo: editting

**Review by eliaporciani — DISMISSED** [review]
LGTM. Just a typo at the beginning

File: search-services/latest/admin/index.md (hunk 1 above)
Comment (oliveralfresco2018) [reply search-services/latest/admin/index.md]: cheers fixed

Comment (klalia) [reply search-services/latest/admin/index.md]: Fixed, thank you

**Review by killerboot — APPROVED** [review]

**Review by robfrank — APPROVED** [review]

**Comment by oliveralfresco2018** [comment]
@robfrank hi Rob, can you check out this conflict and resolve it please?
I'm not sure what the best approach is. cheers.

**Pull request merged**